src/card.css

```diff
@@ -0,0 +1,71 @@
+.card {
+  background-color: var(--white-color);
+  max-width: 200px;
```

Comment:
Try to organize your styles
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)